source/extensions/transport_sockets/tls/ocsp/ocsp.h

```diff
@@ -210,6 +210,25 @@ class OcspResponseWrapper {
    */
   bool isExpired();
 
+  /**
+   * @returns the seconds until this OCSP response expires.
+   */
+  uint64_t secondsUntilExpiration() const;
+
+  /**
+   * @return The beginning of the validity window for this response.
+   */
+  const Envoy::SystemTime getThisUpdate() const;
```

Comment:
top-level const doesn't affect method signature unless it's a reference/pointer.

```suggestion
  Envoy::SystemTime getThisUpdate() const;
```